frontend/src/HomePage/Folders.jsx

```diff
@@ -29,6 +29,18 @@ export const Folders = function Folders({
       });
       return;
     }
+    
+    for(let i = 0; i < folders.length; i++){
```

Comment:
I think we should handle this at the entity level or in DB schema. 
@akshaysasidrn what do you think?

Reply:
@Navaneeth-pk sorry I can't get you, what do I need to change ?

Reply:
Yeah, I agree. 
Basically at the backend at server we need to check if the app is already added to the folder and raise an error if so.

Reply:
ok,shall I work on backend ? @Navaneeth-pk @akshaysasidrn

Reply:
@saisumanthkumar sure 👍

Reply:
@Navaneeth-pk so can I close this PR now

Reply:
@saisumanthkumar no, you can use the same PR. Just revert the old changes and make changes in the backend instead.

Reply:
@Navaneeth-pk ok